composer.json

```diff
@@ -35,12 +35,12 @@
     },
     "require-dev": {
         "mongodb/builder": "^0.2",
-        "laravel/scout": "^11",
+        "laravel/scout": "^10.3",
         "league/flysystem-gridfs": "^3.28",
         "league/flysystem-read-only": "^3.0",
         "phpunit/phpunit": "^10.3",
         "orchestra/testbench": "^8.0|^9.0",
-        "mockery/mockery": "^1.4.4",
+        "mockery/mockery": "^1.4.4@stable",
```

Comment:
This prevent installing unstable release that fails with PHP 8.4

Reply:
I assume this is needed because of `minimum-stability: dev` below. Why does this library have that to begin with?

Reply:
I think that was to test upcoming versions of Laravel. If found issues before the release thanks to that.